### DISCUSSION TITLE
How to increase functions timeout with vercel.json

### DISCUSSION THREAD

**Discussion by juliooa** [vercel/community#1478] [Help] [answered]
### Summary

Hi!
I'm using Sveltekit in the PRO plan, I want to increase the functions timeout to 60, but I can't figure it out where to put the vercel.json file and set the correct route. I keep getting `Error: The pattern "routes/api/**/*.ts" defined in functions doesn't match any Serverless Functions inside the api directory.`

I've tried: 
```
 "functions": {
      "api/**/*.ts": {
        "maxDuration": 60
      }
    }
```
```
 "functions": {
      "routes/api/**/*.ts": {
        "maxDuration": 60
      }
    }
```
```
 "functions": {
      "src/routes/api/**/*.ts": {
        "maxDuration": 60
      }
    }
```

What is the correct path for sveltekit projects?

Thanks!

**Comment by frencojobs** [comment, marked as answer]
You can configure them in `svelte.config.js` like this.
```js
import adapter from '@sveltejs/adapter-vercel'

const config = {
  kit: {
    adapter: adapter({
      maxDuration: 60,
    }),
  },
}

export default config
```

You have to install `@sveltejs/adapter-vercel`, default `adapter-auto` doesn't take or pass these configurations along.

**Reply by juliooa** [reply]
oooh interesting! thanks I didn't know that.

Is it written in the documentation somewhere? for reference.

thanks man!

**Reply by amyegan** [reply]
SvelteKit's [Build and Deploy: Vercel](https://kit.svelte.dev/docs/adapter-vercel) page has some info about configuring [`adapter-vercel`](https://github.com/sveltejs/kit/tree/master/packages/adapter-vercel) to customize your deployment instead of using default settings with [`adapter-auto`](https://kit.svelte.dev/docs/adapter-auto)

**Reply by juliooa** [reply]
got it, thanks!

**Comment by AFU92** [comment]
@frencojobs  I love u 💙 

I used sveltejskit and i was getting this error:

```
Task timed out after 15.00 seconds
Runtime.Unknown

```
```
  kit: {
    paths: {
      relative: false
    },
    adapter: adapter({
      runtime: 'nodejs18.x',
      maxDuration: 60
    })
  }
  ```
  
  Thaaanks 😄
